### ISSUE TITLE
Clean up the final Python 2->3 Syntax incompatibilities

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8461]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8461|
|Type|enhancement|
|Created|2016-06-13 03:07:58Z|
|Branch|https://github.com/twisted/twisted/tree/8461-final-stretch|

This fixes up the last few files to be Python 3 syntax compatible, that is, allowing them to be imported by the Python 3.5 ast module, allowing us to move our linters to Python 3.

<details><summary>Searchable metadata</summary>

```
trac-id__8461 8461
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__8461_final_stretch 8461-final-stretch
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465787278259391 1465787278259391
changetime__1465907040985279 1465907040985279
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Happy with this. Please review.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

You can verify this with ```python3 -m compileall twisted/ -q```.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

I looked at these two patches:

https://github.com/twisted/twisted/commit/cd19fb019d2e7ff8ff70c981e10f7091ce229728
https://github.com/twisted/twisted/commit/74d9526f65ed75e3da9b6b77377a4a4ca3814f8f

and they look OK to me.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_5"></a>|@rodrigc commented|
|-|-|

Looks OK to land.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)|@rodrigc set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 1983d0d3c118e1786ab85f492aa95bdbd9b3f737
```
#!CommitTicketReference repository="" revision="1983d0d3c118e1786ab85f492aa95bdbd9b3f737"
Merge 8461-final-stretch: Clean up the final Python 3 syntax incompatibilities.

Author: hawkowl
Reviewer: rodrigc
Fixes: #8461
```
